v3/src/components/case-table/use-selected-rows.ts

```diff
@@ -119,7 +119,8 @@ export const useSelectedRows = ({ gridRef }: UseSelectedRows) => {
   // anchor row for shift-selection
   const anchorCase = useRef<string | null>(null)
 
-  const handleRowClick = useCallback(({ __id__: caseId }: TRow) => {
+  const handleCellClick = useCallback(
+  ({ row: { __id__: caseId } }: CellClickArgs<TRow>, event: MouseEvent<HTMLDivElement>) => {
```

Comment:
Two things:
1. Where RDG templatized types are concerned, the convention is to add new types to `case-table-types.ts`, so I would add `TCellClickArgs = CellClickArgs<TRow>` there for consistency.
2. Since they're passing in the event now, the `isExtending` definition below should be based on the event. The existing code was written to work around the fact that the event wasn't available in earlier versions of the API.